### PR TITLE
fix minor typos

### PR DESCRIPTION
**Pull request by yashbhutwala**

### CODE REVIEW

File: docs/book/src/cronjob-tutorial/testdata/project/main.go [hunk 1]
```diff
@@ -34,10 +34,10 @@ import (
 
 /*
 The first difference to notice is that kubebuilder has added the new API
-group's package (`kbatchv1beta1`) to our scheme.  This means that we can use those
+group's package (`batchv1`) to our scheme.  This means that we can use those
 objects in our controller.
 
-We'll also need to add the kubernetes batch v1 scheme, since we're creating
+We'll also need to add the kubernetes batch v1 (`kbatchv1beta1`) scheme, since we're creating
```
Comment (mengqiy) [inline docs/book/src/cronjob-tutorial/testdata/project/main.go]: It should be `kbatchv1` here. You please also update the import path to `kbatchv1` in line 23.

Comment (yashbhutwala) [reply docs/book/src/cronjob-tutorial/testdata/project/main.go]: Good catch, done!

**Comment by yashbhutwala** [comment]
@mengqiy I'm not sure why Travis build is failing.  Do you know what this error is about? https://travis-ci.org/kubernetes-sigs/kubebuilder/jobs/559056078#L670

**Pull request merged**